imagersite/imager_profile/views.py

```diff
@@ -1,3 +1,57 @@
 from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import get_object_or_404
+from django.contrib.auth.models import User
+from django.forms import ModelForm
+from django.views.generic.edit import UpdateView
+from .models import ImagerProfile
+
+PROFILE_TEMPLATE = 'profile.html'
+
+
+@login_required(login_url='/login')
+def Profile(request, *args, **kwargs):
+    """Authenticated User Profile."""
+    return render(request, PROFILE_TEMPLATE)
+
+
+def edit_user_profile(request):
+    """Edit user profile."""
+    current_user = User.objects.get(pk=request.user.id)
+    current_profile = current_user.profile
+    if request.method == 'POST':
+        form2 = EditUser(request.POST, instance=current_user)
+        form1 = EditProfile(request.POST, instance=current_profile)
```

Comment:
boy, it would be nice if you didn't have to have an else clause below that had nothing in it but a repeat of exactly these two lines.  Is request.POST nothing at all on GET requests?  Can you just put it in the form initializer anyway?